python/triton_kernels/triton_kernels/matmul_ogs.py

```diff
@@ -526,14 +532,23 @@ def matmul_ogs(x, w, bias,
     w_tensor_or_tma = w_storage.make_tma([1, opt_flags.block_k, opt_flags.block_n], "dense") if w_has_tma else w_storage.data
     # create tma descriptor for w_scale
     w_scale_has_tma = opt_flags.is_persistent and w_scale is not None
+    # When stride(-2) == stride(-1) == 1, it's ambiguous whether W is transposed
+    # (i.e. col-wise). Since this matters when w_has_mx is True and w_transpose
+    # is True the fast code path, stride(-2) == 1 takes precedence, e.g., vs.
+    # w_transpose = w_storage.data.stride()[-1] != 1
     w_transpose = w_storage.data.stride()[-2] == 1
```

Comment:
What about writing it as
```suggestion
    w_transpose = w_storage.data.mT.is_contiguous()
```

Reply:
Actually mT.is_contiguous() is a stronger condition that checks all dims are contiguous vs. here we just want to check dim -2 is contiguous. I believe this was causing CI error. Will revert.